.github/workflows/build.yml

```diff
@@ -120,8 +145,8 @@ jobs:
       - name: Install cibuildwheel
         run: |
           # cibuildwheel does the heavy lifting for us. Originally tested on
-          # 1.8.0, but should be fine at least up to any major new release.
-          python -m pip install 'cibuildwheel<2.0'
+          # 2.3.0, but should be fine at least up to any major new release.
+          python -m pip install 'cibuildwheel<2.4'
```

Comment:
Technically the comment doesn't match the pin any more - you've pinned to a minor version, not a major one.